content/html/concepts/elements/elements.md

```diff
@@ -16,17 +16,49 @@ An HTML element is a piece of content in an HTML document.
 
 ## Syntax
 
+Elements consist of at least one tag but can have two tags with optional content. The tag name determines how the element will be rendered in the HTML document. Elements can also have [attributes](https://www.codecademy.com/resources/docs/html/attributes).
```

Comment:
```suggestion
**Elements** are discrete components within an HTML document that generally consist of at least one tag but can have two tags with optional content. The tag name determines how the element will be rendered in the HTML document. Elements can also have [attributes](https://www.codecademy.com/resources/docs/html/attributes).
```

Reply:
I think this should be the introductory text and should move up to line 17. The  "syntax" header should come after this paragraph, and perhaps we could follow it with a sentence or two to preface the details of each of the element variations.

Reply:
I updated the text accordingly.